server/vcap-services.es6

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+// Environment variables
+
+const vcapConstants = {};
```

Comment:
I've seen a couple of npm packages for vcap services, but i'm wondering if we want to talk to cloud.gov folks to maintain a more official parser?